lang/it.json

```diff
@@ -36,7 +36,7 @@
         "LdapUserAccessAttributeServerSeparator": "Server Attributi Accesso Utente & Separatore Elenco Siti",
         "LdapUserAccessAttributeServerSeparatorDescription": "Stringa usata per separare gli ID delle istanze server dagli ID elenchi siti. Se impostata a ':', ci si aspetterà un attributo di accesso simile a '<server-id>:<site-list>;<server-id>:<site-list>'.",
         "LdapUserAccessAttributeServerSpecDelimiter": "Delimitatore Specifiche Server Attributi Accesso Utente",
-        "LdapUserAccessAttributeServerSpecDelimiterDescription": "Stringa utilizzata per delimitare le specifiche del server in un attributo di accesso utente. Se impostata a ';', ci si aspetta che l'attributo di accesso sia simile a '<server-spec>;<server-spec>;...'.",
+        "LdapUserAccessAttributeServerSpecDelimiterDescription": "Stringa utilizzata per delimitare le specifiche del server in un attributo di accesso utente. Se impostata a ';', ci si aspetta che l'attributo di accesso sia simile a '<server-spec>;<server-spec>;…'.</server-spec></server-spec>",
```

Comment:
@sgiehl Should I still merge this even though isn't adding some HTML that I don't think is supposed to be there?

Reply:
If you see obvious mistakes might be good to fix them on weblate, so they get fixed before merging.

Reply:
@sgiehl I don't think I currently have a Weblate account. Can I just create one?

Reply:
Yes. Sure. Send me the nickname and I can also give you higher permissions.

Reply:
@sgiehl I created a Weblate user `snake14`. I think I made the change, but I'm not sure if it went through.

Reply:
@sgiehl It looks like it didn't work because it's Weblate that's making the HTML change. Note the notification at the top:
![image](https://github.com/matomo-org/plugin-LoginLdap/assets/11394099/e48aec84-6fc6-4016-94bb-0443997afd62)

Reply:
@sgiehl Any idea what issue @snake14 is facing ?

Reply:
This should be fixed now. We can't use text that looks like html tags unless they are html tags. Weblate automatically closes open html tags at the end of a string. I've changed the English original text to use html entities (`&gt;` / `&lt;`) instead and also updated all translations to do so.

Reply:
Thank you @sgiehl . I'll watch out for that in the future.